src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -2009,7 +2019,9 @@ func (s *service) readEncodedResult(
 		s.pools.segmentsArray.Put(segments)
 	}))
 
+	var sp opentracing.Span
 	for _, readers := range encoded {
+		ctx, sp = ctx.StartTraceSpan(tracepoint.FetchReadSegment)
```

Comment:
Nit: just set `ctx, sp := ` inside this block rather than defining the span outside? It should amount to the same GC overhead regardless.

Reply:
The issue w/ this is that I then get a compile error since `ctx` is not used
![image](https://user-images.githubusercontent.com/9585325/74568394-e0bc5c80-4f45-11ea-9ea4-b2a4e9c101a5.png)

which I believe is because it is a newly declared var as opposed to just overwriting the `ctx` arg passed in?

I guess we probably don't want to do anything more w/ the child context, so I think I can just do
```